<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DesignerPreferences</class>
 <widget class="QWidget" name="DesignerPreferences" >
  <property name="objectName" >
   <string notr="true" >DesignerPreferences</string>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="objectName" >
    <string notr="true" />
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QCheckBox" name="optShowDialog" >
     <property name="objectName" >
      <string notr="true" >optShowDialog</string>
     </property>
     <property name="geometry" >
      <rect>
       <x>9</x>
       <y>9</y>
       <width>386</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Show &quot;New Form&quot; Dialog on Startup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxView" >
     <property name="objectName" >
      <string notr="true" >groupBoxView</string>
     </property>
     <property name="geometry" >
      <rect>
       <x>9</x>
       <y>35</y>
       <width>386</width>
       <height>256</height>
      </rect>
     </property>
     <property name="title" >
      <string>Designer UI</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="objectName" >
       <string notr="true" />
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="objectName" >
         <string notr="true" />
        </property>
        <property name="margin" >
         <number>1</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="lblSDI" >
          <property name="objectName" >
           <string notr="true" >lblSDI</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>1</x>
            <y>1</y>
            <width>206</width>
            <height>187</height>
           </rect>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string>SDI picture</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="optSDI" >
          <property name="objectName" >
           <string notr="true" >optSDI</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>1</x>
            <y>194</y>
            <width>206</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text" >
           <string>Multiple Top-Level Windows</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="objectName" >
         <string notr="true" />
        </property>
        <property name="margin" >
         <number>1</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="lblMDI" >
          <property name="objectName" >
           <string notr="true" >lblMDI</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>1</x>
            <y>1</y>
            <width>148</width>
            <height>187</height>
           </rect>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string>MDI Picutre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="optMDI" >
          <property name="objectName" >
           <string notr="true" >optMDI</string>
          </property>
          <property name="geometry" >
           <rect>
            <x>1</x>
            <y>194</y>
            <width>148</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text" >
           <string>All-in-one Window</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
